2-ui/99-ui-misc/01-mutation-observer/article.md

```diff
@@ -207,7 +207,7 @@ let demoElem = document.getElementById('highlight-demo');
 observer.observe(demoElem, {childList: true, subtree: true});
 ```
 
-Here's HTML-element and JavaScript that dynamically fills it using `innerHTML`.
+Here's an HTML-element and JavaScript that dynamically fills it using `innerHTML`.
```

Comment:
Without seeing more context, it's kind of hard, but I would think it should maybe be "Here's an HTML-element and some JavaScript that...". 

The rest of the changes are fine.

Reply:
You can click to the left at these signs to get more context
![image](https://user-images.githubusercontent.com/349336/67151580-bed10700-f2d0-11e9-88f0-43dd6dfe1539.png)

Probably, you already know that.

Reply:
No, just seems obvious after being pointed out.

Reply:
Except that there was nothing I could find to expand down after the part in question, which was what I wanted. 

But I found the article using a "StartPage" search. I now think it's OK to just say "and JavaScript" without any words between. So the changes as provided should be fine.